bundles/org.openhab.binding.fronius/README.md

```diff
@@ -81,7 +81,7 @@ The binding has no configuration options, all configuration is done at `bridge`,
 | `inverterdatadevicestatusstatuscode` | Number                   | Device status code<br />`0` - `6` Startup<br />`7` Running <br />`8` Standby<br />`9` Bootloading<br />`10` Error |
 | `powerflowchannelpgrid`              | Number:Power             | Grid Power (+ from grid, - to grid)                                                                               |
 | `powerflowchannelpload`              | Number:Power             | Load Power (+ generator, - consumer)                                                                              |
-| `powerflowchannelpakku`              | Number:Power             | Battery Power (+ charge, - discharge)                                                                             |
+| `powerflowchannelpakku`              | Number:Power             | Battery Power (- charge, + discharge)                                                                             |
```

Comment:
```suggestion
| `powerflowchannelpakku`              | Number:Power             | Battery Power (+ discharge, - charge)                                                                             |
```

Reply:
My suggested change is just so it's more consistent, i.e. + first followed by - just like the others.

If you agree, you could either use github's feature to apply the suggestion or make the change on your copy and push an update to the branch.

In any case I'm not a main reviewer with merge rights so my suggested change is also subject to a review.